paperqa/agents/main.py

```diff
@@ -121,7 +121,10 @@ async def run_agent(
     else:
         raise NotImplementedError(f"Didn't yet handle agent type {agent_type}.")
 
-    if "cannot answer" in answer.answer.lower() and agent_status != AgentStatus.TIMEOUT:
+    if (
+        "cannot answer" in answer.answer.lower()
```

Comment:
not related to this PR but we should really turn this into a function or module level thing

Reply:
Sure, since it's outside the scope of this PR, I made https://github.com/Future-House/paper-qa/pull/516 to handle this